geometry_msgs/TransformStamped transform
sensor_msgs/CameraInfo camera_info
sensor_msgs/Image depth
morefusion_panda_ycb_video/VoxelGridArray grids
---
sensor_msgs/Image label_ins
